cloudwatch-kinesis/variables.tf

```diff
@@ -0,0 +1,93 @@
+locals {
+  ecs_cluster_name = var.ecs_cluster_name == null ? "${var.project}-${var.environment}" : var.ecs_cluster_name
+  name             = "${var.project}-${var.environment}-http-endpoint"
+  log_group_name   = "kinesis/${local.ecs_cluster_name}"
```

Comment:
@swiknaba should this be /kinesis ? 
I saw we have a pattern to prepend a / in front of the log group but i don't know why. 😆